app/views/admin/resources/_form.html.haml

```diff
@@ -7,9 +7,9 @@
         Resource
 .row
   .col-md-8
-    = semantic_form_for(@resource, :url => (@resource.new_record? ? admin_conference_resources_path : admin_conference_resource_path(@conference.short_title, @resource))) do |f|
```

Comment:
Remove the changes in this file from this PR, the offenses can be solved afterwards :slightly_smiling_face: